Makefile

```diff
@@ -256,7 +256,7 @@ generate-winmanifest:
 ifeq ($(USE_CONTAINER),1)
 	$(RERUN_IN_CONTAINER)
 else
-	go generate ./internal/winmanifest
+	GOOS=linux GOARCH=amd64 go generate ./internal/winmanifest
```

Comment:
Because of our cross compiling this needs to happen with linux amd64 and NOT windows.